src/picongpu/include/plugins/adios/ADIOSWriter.def

```diff
@@ -55,16 +61,16 @@ namespace po = boost::program_options;
 {                                                                              \
     int _err_code = _cmd;                                                      \
     if (_err_code != ADIOS_SUCCESS)                                            \
-        log<picLog::INPUT_OUTPUT > ("ADIOS: error at cmd '%1%': '%2%'") %      \
-            #_cmd % _err_code;                                                 \
+        log<picLog::INPUT_OUTPUT > ("ADIOS: error at cmd '%1%': '%2%' %3%") %  \
```

Comment:
- please use `std:err` that we get the error if no `INPUT_OUTPUT` lvl is set.
- please add line and file info the the error message: `<<__FILE__<<">:"<<__LINE__<<`